bench/irmin-pack/layers.ml

```diff
@@ -296,7 +319,13 @@ let ncommits =
   Arg.(value @@ opt int 4096 doc)
 
 let ncycles =
-  let doc = Arg.info ~doc:"Number of cycles." [ "b"; "ncycles" ] in
+  let doc =
+    Arg.info
+      ~doc:
+        "Number of cycles. This will be in addition to the 5 cycles that run \
```

Comment:
Added this documentation here, hopefully it makes it clearer that it will be 5 + ncycles running. cc @CraigFe @icristescu